<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chi tiết phim</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 #2C3E50, stop:1 #3498DB);
}

QLabel {
    color: white;
    font-weight: bold;
    font-size: 14px;
    padding: 5px;
}

QLineEdit {
    background-color: rgba(255, 255, 255, 0.9);
    border: 2px solid #3498DB;
    border-radius: 4px;
    padding: 5px;
    color: #2C3E50;
    font-size: 14px;
}

QLineEdit:focus {
    border-color: #2980B9;
    background-color: white;
}

QSpinBox, QDoubleSpinBox {
    background-color: rgba(255, 255, 255, 0.9);
    border: 2px solid #3498DB;
    border-radius: 4px;
    padding: 5px;
    color: #2C3E50;
    min-height: 20px;
}

QSpinBox:focus, QDoubleSpinBox:focus {
    border-color: #2980B9;
    background-color: white;
}

QSpinBox::up-button, QDoubleSpinBox::up-button {
    background-color: #2980B9;
    border-radius: 2px;
    width: 16px;
}

QSpinBox::down-button, QDoubleSpinBox::down-button {
    background-color: #2980B9;
    border-radius: 2px;
    width: 16px;
}

QSpinBox::up-button:hover, QDoubleSpinBox::up-button:hover,
QSpinBox::down-button:hover, QDoubleSpinBox::down-button:hover {
    background-color: #3498DB;
}

QPushButton {
    background-color: #2980B9;
    border: 2px solid #2980B9;
    border-radius: 4px;
    color: white;
    padding: 5px 15px;
    font-weight: bold;
    min-width: 80px;
    min-height: 25px;
}

QPushButton:hover {
    background-color: #3498DB;
    border-color: #3498DB;
}

QPushButton:pressed {
    background-color: #1F618D;
    border-color: #1F618D;
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>Tên phim:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="title_edit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_director">
     <property name="text">
      <string>Đạo diễn:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="director_edit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_year">
     <property name="text">
      <string>Năm:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="year_spin">
     <property name="minimum">
      <number>1900</number>
     </property>
     <property name="maximum">
      <number>2100</number>
     </property>
     <property name="value">
      <number>2024</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_rating">
     <property name="text">
      <string>Đánh giá:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="rating_spin">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Hủy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
